docs/deploying-on-k8s.md

```diff
@@ -0,0 +1,74 @@
+# Deploying Carbon-Relay-Ng on Kubernetes
+
+We support two methods of deploying Carbon-Relay-Ng onto Kubernetes, via Tanka and by directly applying the provided
+`.yaml` files.
+
+# Tanka
+
+In this repository in `/ksonnet/lib/carbon-relay-ng` we provide a jsonnet library which uses the [ksonnet
+library](https://github.com/ksonnet/ksonnet-lib) to create a simple carbon-relay-ng deployment. This library also
+depends on the [ksonnet-util library](https://github.com/grafana/jsonnet-libs/tree/master/ksonnet-util) for some helper
+functions.
+
+Using [Tanka](https://tanka.dev/) and [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler), a simple
+carbon-relay-ng deployment can be created like shown here:
+
+Initialize a Tanka environment, this will setup a directory structure and pull in the ksonnet and ksonnet-util
+libraries.
+
+```
+tk init
+```
+
+Install the carbon-relay-ng jsonnet lib:
+
+```
+jb install 'github.com/grafana/carbon-relay-ng/ksonnet/lib/carbon-relay-ng@document_k8s_deployment'
```

Comment:
Before merging, we should:
```suggestion
jb install github.com/grafana/carbon-relay-ng/ksonnet/lib/carbon-relay-ng
```
Left it like this as I don't know when we are intending to merge this.

Reply:
suggest renaming the `ksonnet` dir to `jsonnet`.